connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/PlanGenerationTestSuite.scala

```diff
@@ -1199,6 +1199,34 @@ class PlanGenerationTestSuite extends ConnectFunSuite with BeforeAndAfterAll wit
         "b" -> "avg",
         "*" -> "size",
         "a" -> "count")
+
+    simple
```

Comment:
This doesn't work. You need to construct a single `Dataset` per test, now you are just returning the last one. Can you split this into a bunch of tests?

Reply:
done.